cwl/src/main/scala/cwl/Workflow.scala

```diff
@@ -65,11 +65,11 @@ case class Workflow private(
 
         // TODO: Eurgh! But until we have something better ...
         val womValue = womType.coerceRawValue(inputParameter.default.get).get
-        OptionalGraphInputNodeWithDefault(WomIdentifier(parsedInputId), womType, ValueAsAnExpression(womValue))
+        OptionalGraphInputNodeWithDefault(WomIdentifier(parsedInputId, inputParameter.id), womType, ValueAsAnExpression(womValue), parsedInputId)
```

Comment:
ToL: I'm not sure what it referred to before, but does this satisfy the `TODO: Eurgh` comment above?

Reply:
I think the `Eurgh` is for the `.get.get` just below but I could be wrong